scripts/liquidation.js

```diff
@@ -126,9 +131,32 @@ async function main() {
     console.log(`Borrower balance in XCN after liquidation: ${formatNumber(summary.after.borrowerXcnBalance / 1e18)}`);
     console.log(`Borrower balance locked for supply in XCN: ${formatNumber(summary.after.borrowerXcnLockedForSupply / 1e18)}`);
 
-    const liquidatorProfit = (summary.after.xcnLockedForSupply - amountToRepay) / 1e18;
+    const protocolSeizeShareMantissa = await oXCN.protocolSeizeShareMantissa() / 1e16;
+    const protocolSeizeShareCoefficient = await oXCN.protocolSeizeShareMantissa() / 1e18;
```

Comment:
Using additional variable to keep formulas simple, because:
`protocolSeizeShareMantissa = await oXCN.protocolSeizeShareMantissa() / 1e16; // 2.8, whilst we need 0.028 for calculations`

In javascript:
`2.8 / 1e2 = 0.027999999999999997`